node/lib/vsotask.ts

```diff
@@ -424,8 +424,17 @@ export class TestPublisher {
         if(config) {
             properties['config'] = config;
         }
+		
+		if(runTitle) {
+			properties['runTitle'] = runTitle;
+		}
 
-        for(var i = 0; i < resultFiles.length; i ++) {            
+		if(publishRunAttachments) {
+			properties['publishRunAttachments'] = publishRunAttachments;
+		}			
```

Comment:
we have verified that the default is ON and it continues to publish by default ?